arm-datalake-analytics/account/2016-11-01/swagger/account.json

```diff
@@ -0,0 +1,1580 @@
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the list of Azure Storage containers under the specified Azure Data Lake Analytics account.",
+            "schema": {
+              "$ref": "#/definitions/ListBlobContainersResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DataLakeAnalytics/accounts/{accountName}/StorageAccounts/{storageAccountName}/Containers/{containerName}/listSasTokens": {
+      "post": {
+        "tags": [
+          "Account"
+        ],
+        "operationId": "Account_ListSasTokens",
+        "description": "Gets the SAS token associated with the specified Data Lake Analytics and Azure Storage account and container combination.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Azure resource group that contains the Data Lake Analytics account."
+          },
+          {
+            "name": "accountName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Data Lake Analytics account from which an Azure Storage account's SAS token is being requested."
+          },
+          {
+            "name": "storageAccountName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Azure storage account for which the SAS token is being requested."
+          },
+          {
+            "name": "containerName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Azure storage container for which the SAS token is being requested."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the list of Storage Account SAS tokens for the specified Azure Storage account.",
+            "schema": {
+              "$ref": "#/definitions/ListSasTokensResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DataLakeAnalytics/accounts/{accountName}/DataLakeStoreAccounts/{dataLakeStoreAccountName}": {
+      "get": {
+        "tags": [
+          "Account"
+        ],
+        "operationId": "Account_GetDataLakeStoreAccount",
```

Comment:
@amarzavery I will sync with you on this naming convention to understand better, because the name you are recommending above is more confusing. Today the objects are in the following namespace pattern:
Microsoft.Azure.Management.DataLake.Analytics.DataLake**AnalyticsAccountManagementClient**.Account

Your recommendation would result in something like:
Microsoft.Azure.Management.DataLake.Analytics.DataLakeAnalyticsAccountManagementClient.AnalyticsAccounts, which is redundant and doesn't actually differentiate anything, since you can't do anything with store accounts inside an analytics account client other than add/remove them from underneath the analytics account.